src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RemoteProjectSnapshotProjectEngineFactory.cs

```diff
@@ -39,7 +47,37 @@ internal class RemoteProjectSnapshotProjectEngineFactory : DefaultProjectSnapsho
             }
 
             var remoteFileSystem = new RemoteRazorProjectFileSystem(defaultFileSystem.Root, _filePathNormalizer);
-            return base.Create(configuration, remoteFileSystem, configure);
+            return base.Create(configuration, remoteFileSystem, Configure);
+
+            void Configure(RazorProjectEngineBuilder builder)
```

Comment:
I'm not completely sure if this is the right place to wire this up.

Reply:
Not awful